Tasks/Kvaskov/Task5/Task5.py

```diff
@@ -0,0 +1,82 @@
+import random
+import decimal
+
+
+
+
+k = decimal.Decimal()
+x = str()
+a = []
+
+
+
+
+def check_str(string):
+    number_ch_title = []
+    number_ch_lower = []
+    other_ch = []
+    string = str(input("Input some string in lower and upper case\n"))
+    for i in range(len(string)):
+        if ord(string[i]) in range(65,91):
+            number_ch_title.append(string[i])
+        elif ord(string[i]) in range(97, 123):
+            number_ch_lower.append(string[i])
```

Comment:
how to hammer nails with a microscope))) there is much simpler approach with isupper() function